config/sync/views.view.current_user.yml

```diff
@@ -7,9 +7,8 @@ dependencies:
     - node.type.article
   module:
     - node
-    - user
-id: recent_recipes
-label: 'Recent Recipes'
+id: current_user
```

Comment:
Sorry for the nitpicking, but this naming is a little confusing.

I think simply naming the view `Recipes` is the most telling, inside the view you can have multiple ways of displaying the recipes, e.g. the recent recipes, the current user's recipes etc. Please change the name from `current_user`.